transform/app/transform/utils/common.py

```diff
@@ -629,6 +644,75 @@ def add_tg_fields(df: DataFrame) -> DataFrame:
     return df
 
 
+def harvest_exportation(df: DataFrame, harvested_properties: dict) -> None:
+    """
+    Harvest exportation information from instances within the DataFrame
+    Args:
+        df (DataFrame): Input DataFrame containing instance information.
+        harvested_properties (dict): Dictionary to store harvested properties.
+    Assumptions:
+        - Only the first 10 versions of each instance are harvested; subsequent versions are skipped
+          (approx. 0.2% of data is skipped).
+    For each instance:
+    - Extracted Fields:
+        - URL: URL of the instance.
+        - Type: Type of the instance.
+        - Publication Year: Year of publication from the publication date.
+        - License: License information.
+        - PIDs: Persistent identifiers associated with the instance.
+        - Hosted By: The entity hosting the instance.
+    The extracted information is structured into a list of dictionaries for each instance and stored in
+    'harvested_properties[EXPORTATION]'.
+    Note:
+    - 'i' is used to limit harvesting to the first 10 versions of each instance.
+    """
+    instances_list = df.select(INSTANCE).collect()
+    exportation_column = []
+
+    for instances in instances_list:
+        if instances[INSTANCE]:
+            exportation_row = []
+
+            for i, instance in enumerate(instances[INSTANCE]):
+                if i >= 9:
+                    break
+                else:
```

Comment:
This else is not needed. Please delete it and de-indent the code inside it.

Exact info from Pylint:
`R1723: Unnecessary "else" after "break", remove the "else" and de-indent the code inside it (no-else-break)`

Reply:
I removed the unnecessary `else` after the `break` statement and de-indented the code inside it.